pyodide-build/pyodide_build/pywasmcross.py

```diff
@@ -313,11 +313,15 @@ def handle_command(line, args, dryrun=False):
         # some gcc flags that clang does not support actually
         if arg == "-Bsymbolic-functions":
             continue
-        if arg == "-Wl,-Bsymbolic-functions":
-            continue
-        # breaks emscripten see https://github.com/emscripten-core/emscripten/issues/14460
-        if arg == "-Wl,--strip-all":
-            continue
+        if arg.startswith("-Wl"):
+            arg = arg.replace(",-Bsymbolic-functions", "")
+            # breaks emscripten see https://github.com/emscripten-core/emscripten/issues/14460
+            arg = arg.replace(",--strip-all", "")
+            # wasm-ld does not regconize some link flags
+            arg = arg.replace(",--sort-common", "")
+            arg = arg.replace(",--as-needed", "")
+            if arg == "-Wl":
+                continue
```

Comment:
I think this is a bit confusing. Maybe instead:
```py
    ignored_flags = [
        "-Bsymbolic-functions",
        "-Wl,-Bsymbolic-functions",
        "-Wl,--sort-common",
        "-Wl,--as-needed",
        # breaks emscripten see https://github.com/emscripten-core/emscripten/issues/14460
        "-Wl,--strip-all",
        # threading is disabled for now
        "-pthread",
        # On Mac, we need to omit some darwin-specific arguments
        "-bundle",
        "-undefined",
        "dynamic_lookup",
        # only the main module should link libffi, side modules are able to
        # access libffi functions because it is linked to main module.
        "-lffi",
        # this only applies to compiling fortran code, but we already f2c'd
        "-ffixed-form",
        # This flag is needed to build numpy with SIMD optimization but it
        # causes problems if SIMD isn't enabled
        "-mpopcnt",
    ]


    # Go through and adjust arguments
    for arg in line[1:]:
        if arg in optflags_valid and optflag is not None and arg != optflag:
            # There are multiple contradictory optflags provided, use the one
            # from cflags/cxxflags/ldflags
            continue

        if arg in ignored_flags:
            continue
```

Reply:
I think -Wl need special handle, since user may write `-Wl,-xxx,-yyy,-zzz`, where we want to ignore xxx and yyy but not zzz

Reply:
Got it.

Reply:
Could you add a comment saying that?

Reply:
OK, added